tests/foreman/ui/test_docker.py

```diff
@@ -1279,6 +1280,34 @@ def test_positive_delete(self):
                     self.assertIsNone(
                         self.container.search(compute_resource.name, name))
 
+    # BZ 1266842 is a private bug, so we cannot fetch information about it
+    @stubbed('Unstub when BZ1266842 is fixed')
```

Comment:
The bad thing about doing this is it's not gonna be included on the skip report due to BZ. To fix this we can add a user and password for BZ in order to fetch private information. But as you may be already thinking, this is a future improvement that should be addressed later.